Errors using cmake

## Description
Didn't have issues w/ using cmake previously; now seeing some kind of missing library error

I've previously git cloned and been able to compile [julia](https://github.com/JuliaLang/julia/) from source w/o any problems. Recently, I switched to the dev channel to try out the new linux container apps infrastructure, which involved completely power-washing my pixelbook and setting up the developer mode + dev channel fresh. I then went to use chromebrew like before to install compiler tools to build julia, but now I'm running into this cmake error:
```
cmake: /usr/local/lib64/libstdc++.so.6: version `GLIBCXX_3.4.21' not found (required by cmake)
cmake: /usr/local/lib64/libstdc++.so.6: version `GLIBCXX_3.4.22' not found (required by cmake)
cmake: /usr/local/lib64/libstdc++.so.6: version `CXXABI_1.3.9' not found (required by cmake)
make[1]: *** [/home/chronos/user/julia/deps/mbedtls.mk:34: scratch/mbedtls-2.6.0-gpl/build-configured] Error 1
make[1]: *** Waiting for unfinished jobs....
```
Googling around, it seems like there might be a library mismatch between cmake & stdc++, but I can't quite track down where those libraries are coming from or how I would go about upgrading them. I'm confident I'm on the latest versions of all the compiler package versions since I just barely downloaded/installed everything completely fresh.

Happy to help debug or run any commands that could help track down what might be going on here. Thanks in advance for any help or guidance that can be provided!

## Versions and system information
chromebrew:
```
chronos@localhost ~/julia $ crew --version
0.4.3
```

ChromeOS: `cat /etc/lsb-release`
```
chronos@localhost ~/julia $ cat /etc/lsb-release
CHROMEOS_ARC_ANDROID_SDK_VERSION=25
CHROMEOS_ARC_VERSION=4861648
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={01906EA2-3EB2-41F1-8F62-F0B7120EFD2E}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={01906EA2-3EB2-41F1-8F62-F0B7120EFD2E}
CHROMEOS_RELEASE_BOARD=eve-signed-mpkeys
CHROMEOS_RELEASE_BRANCH_NUMBER=0
CHROMEOS_RELEASE_BUILDER_PATH=eve-release/R69-10820.0.0
CHROMEOS_RELEASE_BUILD_NUMBER=10820
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=69
CHROMEOS_RELEASE_DESCRIPTION=10820.0.0 (Official Build) dev-channel eve 
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=dev-channel
CHROMEOS_RELEASE_VERSION=10820.0.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=10820.0.0
```

Model: Pixelbook

Architecture: `uname -m`
```
x86_64
```

## Addtional information

--- a/packages/cmake.rb
+++ b/packages/cmake.rb
@@ -3,21 +3,21 @@ require 'package'
 class Cmake < Package
   description 'CMake is an open-source, cross-platform family of tools designed to build, test and package software.'
   homepage 'https://cmake.org/'
-  version '3.11.1'
-  source_url 'https://cmake.org/files/v3.11/cmake-3.11.1.tar.gz'
-  source_sha256 '57bebc6ca4d1d42c6385249d148d9216087e0fda57a47dc5c858790a70217d0c'
+  version '3.11.4'
+  source_url 'https://cmake.org/files/v3.11/cmake-3.11.4.tar.gz'
+  source_sha256 '8f864e9f78917de3e1483e256270daabc4a321741592c5b36af028e72bff87f5'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/cmake-3.11.1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/cmake-3.11.1-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/cmake-3.11.1-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/cmake-3.11.1-chromeos-x86_64.tar.xz',
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/cmake-3.11.4-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/cmake-3.11.4-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/cmake-3.11.4-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/cmake-3.11.4-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '0a225df6e0d34cfd7b04f346f1a196af2067f2119ac00b4258ae329436337412',
-     armv7l: '0a225df6e0d34cfd7b04f346f1a196af2067f2119ac00b4258ae329436337412',
-       i686: '8ce6a201ffc9deead5b6e31a98c7fa4ee3d5dc149516aad95ae4ab2b2d3b2276',
-     x86_64: '34fc8e28f407c16ad18952183f9f26daa9a4bb9a1872c043fbdd670eb2795015',
+    aarch64: '12b176c138d6034bb0c9afe305342ec6458e926d30ecf69e249fdc1e1e558c3c',
+     armv7l: '12b176c138d6034bb0c9afe305342ec6458e926d30ecf69e249fdc1e1e558c3c',
+       i686: 'ea3cf4cd98c36144894d10dd6a2551ed8ef0a54f097bbb2d25cfc68f5e9493d9',
+     x86_64: '8c26d1c5236b28b1132aec93957292f28aadb3f6065517b3c223c9fc42b0f9ce',
   })
 
   def self.build

